### ISSUE TITLE
Problems with Reader mode

### ISSUE THREAD

**Issue by jbgzz** [apache/pulsar#16210]
Is there no ACK when using pulsar Reader mode? Because every time a Reader restarts, it reads the message that was read before. If there are multiple Reader instances, each Reader will read the same data, resulting in repeated reads, is this the design?

**Comment by gaozhangmin** [comment]
@jbgzz  Yes, reader interface uses non-durable cursor, it won't store the cursor data.

**Comment by jbgzz** [comment]
So what are the scenarios for the Reader model, which is supposed to be unfriendly to most businesses right now

**Comment by jbgzz** [comment]
> @jbgzz Yes, reader interface uses non-durable cursor, it won't store the cursor data.

So what are the scenarios for the Reader model, which is supposed to be unfriendly to most businesses right now

**Comment by gaozhangmin** [comment]
The reader interface provides Pulsar clients with the low-level abstraction necessary to "manually position" themselves within a topic. 
If you want manually manage offset, reader is a choice
